src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/ProjectSteps.java

```diff
@@ -14,13 +19,194 @@
+    public void validateTheProjectIsCreatedWithSpecifyName() {
+        final String actual = project.getProjectNameOnDashboard();
+        Assert.assertEquals(actual, project.getProjectName(), "Project name match");
+    }
+
+    /**
+     * A project to delete.
+     *
+     * @param projectName the project name
+     */
+    @Given("an existing project named as {string} that user intends to delete")
+    public void anExistingProjectNamedAsThatUserIntendsToDelete(final String projectName) {
+        project.setProjectName(projectName);
+    }
+
+    /**
+     * Open context on settings.
+     */
+    @And("open project's settings")
+    public void openProjectsSettings() {
+        final String projectName = project.getProjectName();
+        project.openProjectSettingsbyName(projectName);
+    }
+
+    /**
+     * Delete project by link.
+     */
+    @When("user click over delete project link")
+    public void userClickOverDeleteProjectLink() {
+        confirm.clickOnDeleteProjectLink();
+        confirm.clickOnDeleteButton();
+    }
+
+    /**
+     * Check if project is still displayed.
+     */
+    @Then("The project no longer appear on projects section")
+    public void theProjectNoLongerAppearOnProjectsSection() {
+        Assert.assertFalse(project.isProjectListedOnMainPage(), "False if project is not listed after deletion");
+    }
+
+    /**
+     * Validate non-existance on active projects.
+     */
+    @And("the project is not present on active project")
+    public void theProjectIsNotPresentOnActiveProject() {
+        project.loadMainPage(PROJECTURI);
+        final boolean actual = project.isProjectListedOnProjectPage();
+        Assert.assertFalse(actual, "Passed if project is no longer on active project list");
+    }
+
+    /**
+     * Save on context value of project name.
+     *
+     * @param projectName the project name
+     */
+    @Given("an existing project known as {string} that user intends to edit")
+    public void anExistingProjectKnownAsThatUserIntendsToEdit(final String projectName) {
+        project.setProjectName(projectName);
+        project.openProjectSettingsbyName(projectName);
+    }
+
+    /**
+     * Intend to change values by given attributes.
+     *
+     * @param projectAttributes table of attributes
+     */
+    @And("change values on form as")
+    public void changeValuesOnFormAs(final Map<String, String> projectAttributes) {
+        project.setValuesOnEditProjectForm(projectAttributes);
+        project.saveFormOnEditProject();
+    }
+
+    /**
+     * Save button is pressed.
+     */
+    @When("user press save button")
+    public void userPressSaveButton() {
+        project.saveFormOnEditProject();
+    }
+
+    /**
+     * Verifies is success message type was displayed on screen.
+     */
+    @Then("A successful message is displayed")
+    public void aSuccessfulMessageIsDisplayed() {
+        final boolean actual = project.getResponseMessage();
```

Comment:
please use Toast page instead of Project

Reply:
Yes, the project needs to handle toast pages, we can work on Monday to solved this.